src/display/text_layer.js

```diff
@@ -104,10 +105,10 @@ var renderTextLayer = (function renderTextLayerClosure() {
       left = tx[4] + (fontAscent * Math.sin(angle));
       top = tx[5] - (fontAscent * Math.cos(angle));
     }
-    textDiv.style.left = left + 'px';
-    textDiv.style.top = top + 'px';
-    textDiv.style.fontSize = fontHeight + 'px';
-    textDiv.style.fontFamily = style.fontFamily;
+    textDivProperties.style = 'left: ' + left + 'px; top: ' + top +
+                              'px; font-size: ' + fontHeight +
+                              'px; font-family: ' + style.fontFamily + ';';
```

Comment:
For a good case, such as the tracemonkey paper, there seem to be around 150 divs per page. So even for well behaved documents, it might make sense to avoid these intermediate strings when considering an entire document.

For a really bad case, such as the PDF from issue 7224, there's around 2000 divs *per page*, and in this case it probably makes a lot of sense to avoid intermediate strings.

I'll update the code, and add a comment about why we're doing this.